python/tvm_ffi/utils/_build_optional_torch_c_dlpack.py

```diff
@@ -30,13 +30,6 @@
 import torch.torch_version
 import torch.utils.cpp_extension
```

Comment:
also document at the top. # Important: to avoid cyclic dependency, we avoid import tvm_ffi names at top level